lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
```

Comment:
I'm un-convinced you need multiple test class functions - But give them better non-abbrievated names and you can kill your comments about why they exist